src/axom/klee/Geometry.cpp

```diff
@@ -6,29 +6,136 @@
 #include "axom/klee/Geometry.hpp"
 
 #include "axom/klee/GeometryOperators.hpp"
+#include "conduit_blueprint_mesh.hpp"
 
 #include <utility>
 
 namespace axom
 {
 namespace klee
 {
-bool operator==(const TransformableGeometryProperties &lhs,
-                const TransformableGeometryProperties &rhs)
+bool operator==(const TransformableGeometryProperties& lhs,
+                const TransformableGeometryProperties& rhs)
 {
   return lhs.dimensions == rhs.dimensions && lhs.units == rhs.units;
 }
 
-Geometry::Geometry(const TransformableGeometryProperties &startProperties,
+Geometry::Geometry(const TransformableGeometryProperties& startProperties,
                    std::string format,
                    std::string path,
                    std::shared_ptr<GeometryOperator const> operator_)
   : m_startProperties(startProperties)
   , m_format(std::move(format))
   , m_path(std::move(path))
+  , m_levelOfRefinement(0)
   , m_operator(std::move(operator_))
 { }
 
+Geometry::Geometry(const TransformableGeometryProperties& startProperties,
+                   const axom::sidre::Group* meshGroup,
+                   const std::string& topology,
+                   std::shared_ptr<GeometryOperator const> operator_)
+  : m_startProperties(startProperties)
+  , m_format("blueprint-tets")
+  , m_path()
+  , m_meshGroup(meshGroup)
+  , m_topology(topology)
+  , m_levelOfRefinement(0)
+  , m_operator(std::move(operator_))
+{
+#ifdef AXOM_DEBUG
+  #if 0
```

Comment:
Should this code block be removed?

Reply:
See issue https://github.com/LLNL/axom/issues/1445